docs/en-US/component/notification.md

```diff
@@ -86,7 +86,7 @@ In this case you should call `ElNotification(options)`. We have also registered
 | message                  | description text                                                                                                   | string/Vue.VNode | —                                           | —         |
 | dangerouslyUseHTMLString | whether `message` is treated as HTML string                                                                        | boolean          | —                                           | false     |
 | type                     | notification type                                                                                                  | string           | success/warning/info/error                  | —         |
-| iconClass                | custom icon's class. It will be overridden by `type`                                                               | string           | —                                           | —         |
+| icon                     | custom icon component. It will be overridden by `type`                                                             | string           | —                                           | —         |
```

Comment:
You can change the `string` type to `Vue.Component`

Reply:
updated